client/my-sites/plans/current-plan/jetpack-checklist/index.tsx

```diff
@@ -241,6 +296,7 @@ class JetpackChecklist extends PureComponent< Props > {
 									? `/media/${ siteSlug }`
 									: `/settings/performance/${ siteSlug }`
 							}
+							id={ 'video-hosting' }
```

Comment:
Nitpick, for string literal props in JSX, it's preferable to use 'HTML-style' attribute notation:

```suggestion
							id="video-hosting"
```

(I'll change while rebasing)

Reply:
> (I'll change while rebasing)

Noted and thank you for picking this up. I'll be travelling for the design meetup today but let me know if you have any questions or when would be a good time to check it out again.